ipaserver/plugins/groupfas.py

```diff
@@ -17,30 +17,40 @@
 from ipaserver.plugins.group import group_show
 from ipaserver.plugins.internal import i18n_messages
 
-group.possible_objectclasses.append("fasgroup")
-if "objectclass" not in group.default_attributes:
-    group.default_attributes.append("objectclass")
+from .fasutils import URL
 
-default_attributes = [
+if "fasgroup" not in group.possible_objectclasses:
+    group.possible_objectclasses.append("fasgroup")
+
+group_fas_attributes = [
     "fasurl",
     "fasmailinglist",
     "fasircchannel",
 ]
-group.default_attributes.extend(default_attributes)
+group.default_attributes.extend(group_fas_attributes)
+# always fetch objectclass so group_show can show fasgroup property
+group.default_attributes.append("objectclass")
+
+group.managed_permissions.update(
+    {
+        "System: Read FAS group attributes": {
+            "replaces_global_anonymous_aci": True,
+            "ipapermbindruletype": "all",
+            "ipapermright": {"read", "search", "compare"},
+            "ipapermtargetfilter": ["(objectclass=fasgroup)"],
+            "ipapermdefaultattr": set(group_fas_attributes),
+        },
+    }
+)
 
 group.takes_params += (
     Flag(
         "fasgroup?",
         label=_("FAS group"),
         flags={"virtual_attribute", "no_create", "no_update", "no_search"},
     ),
-    Str(
-        "fasurl*",
-        cli_name="fasurl",
-        label=_("Group URL"),
-        maxlength=255,
-    ),
-    Str(
+    URL("fasurl*", cli_name="fasurl", label=_("Group URL"), maxlength=255,),
+    URL(
```

Comment:
Oh, I was thinking of putting only the mailing list's email address here, not the URL to the archives.